protonvpn_cli/cli.py

```diff
@@ -326,20 +326,35 @@ def configure_cli():
 
         user_choice = user_choice.lower().strip()
         if user_choice == "1":
-            set_username_password(write=True)
+            try:
+                set_username_password()
+                time.sleep(1)
+            except KeyboardInterrupt:
+                print()
```

Comment:
I didn't like the "enter username or type menu to go back to menu" prompt (also, what's with that one guy who's username is "menu"? :D)

I think this is a better approach. pressing CTRL+C throws you back to the menu now. Only in this option, as the others have proper "multiple choice" interactions.